src/test/resources/com/puppycrawl/tools/checkstyle/utils/checkutil/InputCheckUtilTest.java

```diff
@@ -67,4 +67,8 @@ public void setField(Map<String, Integer> field) {
     }
 
     public void testReceiver(InputCheckUtilTest<V, C>this, int variable) {}
+
+    public interface Example {
+        void method();
```

Comment:
please move all unique case from http://rveach.no-ip.org/checkstyle/regression/reports/283/ to Inputs,
Please create special Input file with config in comment on top.

Reply:
@romani this regression run has no config besides the default config.

Reply:
Done.

Reply:
> Please create special Input file with config in comment on top.

I do not see it.
I think simple new Input file creation is better than extending existing, when we know that it is used only in one test.